virttest/_wrappers.py

```diff
@@ -0,0 +1,47 @@
+""" The file contains some wrappers of common python base-package workflows
+    that can be repeated throughout the avocado-vt code.
+    Please, remember that it is a private module.
+    Please, use this module ONLY from other virttest/avocado-vt modules.
+"""
+
+import sys
+import importlib
+
+
+def import_module(name, directory=None):
+    """ Imports a module named <name> to the execution environment.
+        If the module is not in the PYTHONPATH or the local directory,
+        its directory can be determined by the <directory> argument.
+        :param name: Name of the module that is going to be imported
+        :type name: String
+        :param directory: Path of the directory in which the module is located.
+                          If None, it will find the module in the current dir
+                          and the PYTHONPATH.
+        :type directory: String or None
+        :returns: The imported module
+    """
+    if directory:
+        sys.path.insert(0, directory)
+    # Just in case the module source has been dropped after
+    # the interpreter execution had begun
+    importlib.invalidate_caches()
+    module = importlib.import_module(name)
+    if directory:
+        sys.path.pop(0)
```

Comment:
To ensure the given directory must be removed from `sys.path` after importing a module it seems the above two lines have to be put into a `finally` clause, for example:
```python
try:
    importlib.invalidate_caches()
    module = importlib.import_module(name)
finally:
    if directory:
        sys.path.pop(0)
```

Reply:
Adding this now, thank you for yet another catch! :)